requirements.txt

```diff
@@ -0,0 +1,53 @@
+annotated-types==0.7.0
```

Comment:
woah woah woah, you can be using all this shit, I bet 80% of this is currently not used in this codebase, remove the things you are not using